### PR TITLE
Output HTML and Markdown

### PR DESCRIPTION
**Pull request by dmundra**
Closes https://github.com/GSA/openacr/issues/119

This is an initial version that recreate the handlebar templates in Svelte for Markdown and HTML. Will have to add to it when we add the chapters and table fields. For Markdown had to build up a string manually to get the correct spacing and line breaks.

QA steps:

1. After deployment ('Deploy' action) is successful, open site https://gsa.github.io/openacr-editor/.
2. Click 'Start new report'.
3. Switch to the 'About' page.
4. Type in values and they can optionally match https://github.com/GSA/openacr/blob/main/openacr/drupal-9.yaml.
4. Click the 'View report' or 'Report' tab.
5. Confirm you see a 'Valid' message.
6. Confirm the HTML on the page displays the various sections except tables, chapters, related OpenACRs. Use https://federalist-02947dd8-86df-467a-b2af-4c5b94c5b1f0.app.cloud.gov/site/gsa/opat-website/openacr/drupal-9.html as an example to compare against.
7. Click the 'Download Report (HTML)' and confirm the HTML file downloaded matches the page.
8. Click the 'Download Report (Markdown)' and confirm the Markdown file downloaded matches the page.

### CODE REVIEW

File: src/utils/cleanUpReportHTML.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ export function cleanUp(node) {
   });
 
   removeAttributeEls.forEach((el) => {
-    el.removeAttribute("id");
-    el.removeAttribute("class");
+    // el.removeAttribute("id");
```
Comment (mgifford) [inline src/utils/cleanUpReportHTML.js:45]: Why is this commented out? Fine with commenting out in the commit, but curious why.

**Review by mgifford — CHANGES_REQUESTED** [review]
Just had the one comment about the commented out section. Looks pretty good to me. Went through the test. All looked fine. Nice to be able to export the results from the About page.

File: src/utils/cleanUpReportHTML.js (hunk 1 above)
Comment (dmundra) [reply src/utils/cleanUpReportHTML.js:45]: That code removes IDs and classes when it exported the HTML. We need those for our anchor links so I commented it out. We might use the cleanup code to do other cleanup if needed.

**Review by mgifford — APPROVED** [review]
Looks good. Thanks for explaining why those lines were commented out.

**Pull request merged**